src/main/java/com/cgdp/library/dto/author/CreateAuthorResponseDTO.java

```diff
@@ -0,0 +1,6 @@
+package com.cgdp.library.dto.author;
+
+import lombok.Data;
+
+
+public record CreateAuthorResponseDTO(Long id, String name) {}
```

Comment:
Add annotations to check if `id` is null and if `name` is null or empty, also rename `name` to `authorName`
```suggestion
public record CreateAuthorResponseDTO(@NotNull Long id, @NotEmpty String authorName) {}
```
Not sure if those are the correct annotations please check them

Reply:
Update here with `@NotNull` and `@NotBlank`